atlp-rwanda/e-comm-team-brogrammers-bn
# #184581354 tracking order status

**What does this PR do?**
tracking order status 
**Description of Task to be completed?**
buyer should be able to track  order status and admin should be able to update status then after real time update should be sent to the front end.
**How should this be manually tested?**
- npm i && npm run migrate:all
- to  get the status of the buyer here token must be provided 
- http://localhost:5000/order/status/
- admin to update the status , his or her token should be provided and order id that you want to update 
- http://localhost:5000/order/(orderId)
- testing realtime update
- http://localhost:5000/public/sse-test.html
**Any background context you want to provide?**
n/a
What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/story/show/184581354

**Screenshots (if appropriate)**
<img width="1440" alt="Screen Shot 2023-04-06 at 17 22 25" src="https://user-images.githubusercontent.com/99407107/230425621-9a795d47-256c-4637-8a05-5431505bc134.png">
<img width="1440" alt="Screen Shot 2023-04-06 at 17 23 54" src="https://user-images.githubusercontent.com/99407107/230426033-fb2f2cd3-4baf-444c-a15e-37374e043266.png">
<img width="1440" alt="Screen Shot 2023-04-06 at 17 24 07" src="https://user-images.githubusercontent.com/99407107/230426083-ff08d3e0-3374-405f-99af-de05d39f904d.png">


Questions:
